parsers/ERP_PGCB.py

```diff
@@ -166,7 +164,7 @@ def query(
             parser="BD.py",
             message=("Could not find table header in returned HTML."),
         )
-    verify_table(table_head)
+    verify_table_header(table_head)
```

Comment:
This does cause some type errors as the table head can be of Tag | NavigatableString | Int but I don't think it will cause any runtime issues as it is right now.